bluemira/base/config.py

```diff
@@ -1,427 +0,0 @@
-# bluemira is an integrated inter-disciplinary design tool for future fusion
-# reactors. It incorporates several modules, some of which rely on other
-# codes, to carry out a range of typical conceptual fusion reactor design
-# activities.
-#
-# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
-#                    D. Short
-#
-# bluemira is free software; you can redistribute it and/or
-# modify it under the terms of the GNU Lesser General Public
-# License as published by the Free Software Foundation; either
-# version 2.1 of the License, or (at your option) any later version.
-#
-# bluemira is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
-
-"""
-Configuration classes
-"""
-from bluemira.base.config_schema import ConfigurationSchema
-from bluemira.base.parameter import ParameterFrame
-
-
-class Configuration(ConfigurationSchema, ParameterFrame):
-    """
-    The base object for all variable names and metadata in bluemira.
-    Variables specified here should be physical in some way, and not represent
-    how the code is being run.
-    Defaults are also specified here, and overridden later.
-    New variables should be defined here, with a corresponding entry in the
-    ConfigurationSchema, and passed onwards as Parameter objects.
-    """
-
-    # fmt: off
-    params = [
-        ['Name', 'Reactor name', 'Cambridge', 'dimensionless', None, 'Input'],
```

Comment:
Are the descriptions and info here put anywhere at the moment?

Reply:
No, nowhere at the moment. I'm happy for it to just live in the git history until we need it. Unless you think we should put it somewhere?

Reply:
No we dont need to put it somewhere now just making sure I was aware :)